frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -0,0 +1,48 @@
+import { LemonButton, LemonModal, LemonTextArea } from '@posthog/lemon-ui'
+import { billingProductLogic } from './billingProductLogic'
+import { useActions, useValues } from 'kea'
+import { BillingProductV2Type } from '~/types'
+import { billingLogic } from './billingLogic'
+
+export const UnsubscribeSurveyModal = ({ product }: { product: BillingProductV2Type }): JSX.Element | null => {
+    const { surveyID, surveyResponseValue } = useValues(billingProductLogic({ product }))
+    const { setSurveyResponseValue, setSurveyID, reportSurveySent } = useActions(billingProductLogic({ product }))
+    const { deactivateProduct } = useActions(billingLogic)
+
+    const textAreaNotEmpty = surveyResponseValue.length > 0
+    return (
+        <LemonModal
+            onClose={() => {
+                setSurveyID('')
+            }}
+            title="Let us know why you're unsubscribing"
+        >
+            <div className="flex flex-col">
+                <LemonTextArea
+                    placeholder={'Start typing...'}
+                    value={surveyResponseValue}
+                    onChange={setSurveyResponseValue}
+                />
+                <div className="flex justify-between pt-2 gap-8">
+                    <LemonButton
+                        type={'tertiary'}
+                        status={'muted'}
+                        to="mailto:sales@posthog.com?subject=Issues%20With%20Bill"
+                    >
+                        Big bills got you down? Chat with support
```

Comment:
Was trying to be humorous here. Happy to do something more straightforward and less leading like what you had originally suggested @raquelmsmith : `Issues with your bill? Get in touch`